fea-rs/src/compile/error.rs

```diff
@@ -102,19 +102,8 @@ impl std::fmt::Debug for DiagnosticSet {
     }
 }
 
-impl From<ValidationReport> for BinaryCompilationError {
-    fn from(src: ValidationReport) -> BinaryCompilationError {
+impl From<write_fonts::error::Error> for BinaryCompilationError {
+    fn from(src: write_fonts::error::Error) -> BinaryCompilationError {
         BinaryCompilationError(src)
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn assert_compiler_error_is_send() {
```

Comment:
Interesting that we lose `Send` on the error. Was this check necessary?

Reply:
I wish there was a comment; it's hard to tell if it matters. Since this is blocking me right now I'd prefer to push forward and then if we really need Send we can bring it back later.

Reply:
@cmyr please advise when you have a chance.

Reply:
apparently fontmake-rs would like it if it was

Reply:
For the curious, the discussion that prompted this test is at https://github.com/cmyr/fea-rs/pull/119#discussion_r1084797298